standard/expressions.md

```diff
@@ -1025,6 +1070,19 @@ In case the parameter type sequences `{P₁, P₂, ..., Pᵥ}` and `{Q₁, Q₂
 - Otherwise if one member is a non-lifted operator and the other is a lifted operator, the non-lifted one is better.
 - If neither function member was found to be better, and all parameters of `Mᵥ` have a corresponding argument whereas default arguments need to be substituted for at least one optional parameter in `Mₓ`, then `Mᵥ` is better than `Mₓ`. Otherwise, no function member is better.
 
+#### §better-argument-passing-mode-new-clause Better argument-passing mode
+
+It is permitted to have corresponding parameters in two overloaded methods differ only by parameter-passing mode provided one of the two parameters has value-passing mode, as follows:
+
+```csharp
+public static void M1(int p1) { … }
+public static void M1(in int p1) { … }
+```
+
+Given `int i = 10;`, according to [§12.6.4.2](expressions.md#12642-applicable-function-member), the calls `M1(i)` and `M1(i + 5)` result in both overloads being applicable. In such cases, the method with the parameter-passing mode of value is the ***better parameter-passing mode choice***.
```

Comment:
The title of the subsection is better argument-passing mode - should it actually be "better parameter-passing mode"? (I suspect it should.)

Reply:
Leaving unresolved for committee discussion.

Reply:
@BillWagner: We agreed to change the title to "better parameter-passing mode". Let's do that in this PR.